sopel_channelmgnt/channelmgnt/__init__.py

```diff
@@ -451,7 +452,7 @@ def fyckb(bot, trigger):
         if mask == '':
             mask = nick + '!*@*'
         if trigger.account in chanops:
-            bot.write(['MODE', channel, '+b', mask + '$' + forwardchan])
+            bot.write(['MODE', channel, '+b', mask + '${}'.format(bot.settings.channelmgnt.support_channel)])
```

Comment:
Don't use format. Not sure if it's enabled on this repo but the whole of core uses f strings.

Reply:
How?